storage/filesystem/internal/dotgit/dotgit_test.go

```diff
@@ -216,7 +260,92 @@ func (s *SuiteDotGit) TestPackfile(c *C) {
 	}
 }
 
+func (s *SuiteDotGit) TestObjectfiles(c *C) {
+	objectfiles := func(d *DotGit) (fs.FS, []core.Hash, error) {
+		return d.Objectfiles()
+	}
+	for _, test := range [...]struct {
+		fixture string
+		fn      getObjectsHashesFn
```

Comment:
Same as above, why do you need this field?, can you just call the Objecfiles method directly?